### DISCUSSION TITLE
Issue plotting

### DISCUSSION THREAD

**Discussion by liamjh98** [fispact/pypact#51] [Q&A]
When I try and run the example plot functions I get this error: 

module 'pypact.analysis' has no attribute 'get_all_isotopes_states'

**Comment by thomasms** [comment]
Hi! Thanks for reporting this issue.

Are these examples coming from the [`examples`](https://github.com/fispact/pypact/tree/master/examples) folder or from the README?

If you could kindly share the full example/script so I can reproduce that would be much appreciated.

Also, if you could state which version you're using, that would also be helpful.

**Comment by liamjh98** [comment]
![image](https://github.com/fispact/pypact/assets/72528278/81ffe6cf-abcb-481e-b5cb-a6f0a0a15971)

**Reply by liamjh98** [reply]
Python version 3.11.6

**Reply by thomasms** [reply]
Thanks for sharing. I will look into this and get back to you

**Reply by thomasms** [reply]
Sorry, I meant what version of pypact are you using?

**Comment by thomasms** [comment]
Fixed in version 1.3.7.

Try upgrading pypact via:

```bash
pip3 install --upgrade pypact 
```
